src/Components/Movies/Movies.js

```diff
@@ -38,9 +38,10 @@ class Movies extends Component {
 
 
   render() {
+    let moviesDisplay = this.displayAllMovies();
     return (
-      <section className="movies-container">
-        {this.displayAllMovies()}
+      <section className="movies-container" placeholder="mov-card">
```

Comment:
I think this is actually a situation where a `testid` would be more appropriate. The `placeholder` attribute should only be used on inputs.

Reply:
Actually you should be able to use getByRole with the [`region` role](https://www.w3.org/TR/html-aria/#el-section)!